internal/config/index.go

```diff
@@ -37,6 +37,12 @@ type Indexer struct {
 	// AutoIndexDurationLimit represents auto indexing duration limit
 	AutoIndexDurationLimit string `yaml:"auto_index_duration_limit" json:"auto_index_duration_limit"`
 
+	// AutoSaveIndexDurationLimit represents auto save index duration limit
+	AutoSaveIndexDurationLimit string `yaml:"auto_save_index_duration_limit" json:"auto_save_index_duration_limit"`
+
+	// AutoSaveIndexWaitDuration represents auto save index wait for next duration
+	AutoSaveIndexWaitDuration string `yaml:"auto_save_index_wait_for_next_duration" json:"auto_save_index_wait_for_next_duration"`
```

Comment:
Why is the name different for field and yaml? It is better to use same name.
I think `AutoSaveIndexWaitDuration` is more understandable.

Reply:
ok, renamed.